src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState, useMemo } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getTodosSelector } from "../../store";
+import { fetchTodos } from "../../store/actions";
+
+import { Todo } from "../Todo/Todo";
+
+import "./TodoList.scss";
+
+const options = ["all", "active", "completed"];
```

Comment:
move to constants file, and turn it to enum

Reply:
done